consensus/tests/miner.rs

```diff
@@ -71,4 +82,52 @@ mod miner {
         let parent_header = genesis().header.into();
         test_find_block(&transactions, &parent_header).await;
     }
+
+    #[tokio::test(flavor = "multi_thread")]
+    async fn terminate_on_block() {
+        tracing_subscriber::fmt::init();
```

Comment:
```suggestion
```
(in order not to get a panic if there are any other tests initializing `tracing_subscriber` in this crate in the future